Generals/Code/GameEngine/Source/GameClient/GUI/LoadScreen.cpp

```diff
@@ -193,6 +194,19 @@ SinglePlayerLoadScreen::~SinglePlayerLoadScreen( void )
 
 }
 
+Bool SinglePlayerLoadScreen::isVideoPlaying( void ) const
```

Comment:
Do we need this function even at all? We could make `skipVideo` return Bool instead and use that to detect if it was skipped.

```cpp
Bool GameLogic::skipLoadScreen( void )
{
  return m_loadScreen && m_loadScreen->skipVideo();
}
```